src/main/scala/beam/router/BeamSkimmer.scala

```diff
@@ -247,12 +249,17 @@ class BeamSkimmer @Inject()(val beamConfig: BeamConfig, val beamServices: BeamSe
     ProfilingUtils.timed(s"writeCarSkimsForPeakNonPeakPeriods on iteration ${event.getIteration}", x => logger.info(x)) {
       writeCarSkimsForPeakNonPeakPeriods(event)
     }
+    ProfilingUtils.timed(s"writeObservedSkimsPlus on iteration ${event.getIteration}", x => logger.info(x)) {
```

Comment:
Note that `x => logger.info(x)` can be just the method reference `logger.info`

Reply:
Done